.github/actions/native-models/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Generate native models'
+description: 'Generate native models from thrift schema'
+inputs:
+  personal-access-token:
+    description: 'Personal access token used to interact with repo'
+    required: true
+  branch-name:
```

Comment:
I thought this input was useful for now, it allows us to generate models for any branch, including development branches. The result will be a commit to the named branch with the latest models.